### DISCUSSION TITLE
The new automatic refresh of notifications is broken in multiple ways

### DISCUSSION THREAD

**Discussion by peppy** [community/community#10276] [Mobile] [answered]
The notifications screen now refreshes on returning to it (testflight). This has multiple issues and should be rolled back until they can be fixed, as it makes the app unusable under normal circumstances.

- When dealing with one notification (ie. replying) then coming back to the list to continue working through a backlog, if any notifications above the current view position have been updated or added, the list scrolls away and the user loses their current position. (haven't caught this on video yet)

- When swiping a notification "done" after exiting the notification info view, the refresh will cause the notification to reappear after it has disappeared:

https://user-images.githubusercontent.com/191335/150053938-36e6ae31-4507-4313-917a-2bd26b40355e.MOV

**Comment by brianlovin** [comment, marked as answer]
Thanks for the nudge @peppy — we're going to revert these changes in the next update, and will explore better ways to implement the change so that these issues don't happen in the future!
